core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/SnowflakeCallMapper.scala

```diff
@@ -50,6 +51,7 @@ class SnowflakeCallMapper extends ir.CallMapper with ir.IRHelpers {
       case ir.CallFunction("IFNULL", args) => ir.Coalesce(args)
       case ir.CallFunction("IS_INTEGER", args) => isInteger(args)
       case ir.CallFunction("JSON_EXTRACT_PATH_TEXT", args) => getJsonObject(args)
+      case ir.CallFunction("LAST_DAY", args) => parse_last_day(args)
```

Comment:
Please use camelCase in Scala
```suggestion
      case ir.CallFunction("LAST_DAY", args) => parseLastDay(args)
```

Reply:
Done